src/components/tabs/tab.js

```diff
@@ -39,6 +39,9 @@ export const BTab = /*#__PURE__*/ Vue.extend({
       type: [String, Array, Object],
       default: null
     },
+    titleLinkAttributes: {
+      type: Object
+    },
```

Comment:
Add in `default: null` to coerce `undefined` into `null`

Reply:
This should be the last tweak needed

Reply:
And the `package.json` file in the component's directory will need updating to include this new prop for `BTab` section.